packages/devtools_app/lib/src/inspector/inspector_tree_controller.dart

```diff
@@ -542,6 +585,151 @@ class InspectorTreeController extends Object {
       }
     }
   }
+
+  /* Search support */
+  @override
+  void previousMatch() {
+    super.previousMatch();
+    final currentSearchMatches = searchMatches.value;
```

Comment:
why can't we use `matchIndex` from `SearchControllerMixin` for this to grab the current match? 

More generally, perhaps instead of overriding `previousMatch` and `nextMatch`, we should have another method, `SearchControllerMixin.onMatchChanged` that `SearchControllerMixin.previousMatch` and `SearchControllerMixin.nextMatch` can call, and subclasses can override and implement `onMatchChanged` instead.

Reply:
`onMatchChanged()` has been added